docker/kitsune2_bootstrap_srv_auth_test/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM rust:1.85-bookworm AS builder
+
+# Pick a directory to work in
+WORKDIR /usr/src/kitsune2
+
+# Copy the workspace's source code into the container
+ADD ["Cargo.toml", "Cargo.lock", "./"]
+ADD crates ./crates
+
+# Build the bootstrap server
+RUN cargo install --path ./crates/bootstrap_srv --example test-auth-hook-server
+RUN cargo install --path ./crates/bootstrap_srv --bin kitsune2-bootstrap-srv
```

Comment:
Because of the comment below, I think this image should just contain the auth server and the boostrap server image can be deployed alongside

Reply:
Done: https://github.com/holochain/kitsune2/pull/212/commits/a2b1a1966dcc9688893074dae0546792cedd6cbc